### PR TITLE
doc: update license copyright to OpenJS foundation

### PR DESCRIPTION
**Pull request by MylesBorins**
This is a relic of the repo being created in the Node.js foundation org

My assumption is this should be non controversial but want a gut check.

Closes: https://github.com/openjs-foundation/cross-project-council/issues/382

### CODE REVIEW

**Comment by MylesBorins** [comment]
We should get feedback from foundation folks before fast tracking this

**Comment by molant** [comment]
In this conversation with @brianwarner we talked about this same issue for webhint. it's OK to replace with openjs:

https://github.com/openjs-foundation/cross-project-council/pull/414#discussion_r353510795

**Comment by brianwarner** [comment]
In this particular instance, I would recommend adding "OpenJS Foundation" in addition to "Node.js contributors."

Here's my rationale, since @molant referenced the other thread and I'd like to avoid creating any confusion in this response.  The general guidance is that the text "JS Foundation" and "Node.js Foundation" can be updated to "OpenJS Foundation", since it is documenting a fact that the two orgs are merged and renamed.

Here, the line just says "Node.js contributors".  Since that could mean either project or Foundation, I would recommend the almost-always-safe path of appending rather than replacing.

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License (MIT)
 
-Copyright (c) Node.js contributors. All rights reserved.
+Copyright (c) OpenJS Foundation and contributors. All rights reserved.
```
Comment (brianwarner) [inline LICENSE.md]:
```suggestion
Copyright (c) Node.js contributors. All rights reserved.
Copyright (c) OpenJS Foundation and contributors. All rights reserved.
```

**Comment by summerrobert7281** [comment]
Wasn't it spelled differently though like fastrak or something..??

**Pull request merged**